### DISCUSSION TITLE
Adding Gemini as translator backend

### DISCUSSION THREAD

**Discussion by dgoryeo** [Cerlancism/chatgpt-subtitle-translator#65] [General]
Gemini would be a good fit for Cerlancism translator since it provides much larger window size than GPT. Would you consider to add Gemini 1.5 pro as an engine?

**Comment by Cerlancism** [comment]
You should able to do it by just overriding the OPENAI_BASE_URL env

Documentation from Google appears to provide such endpoints:

https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/call-vertex-using-openai-library

**Comment by Cerlancism** [comment]
Or run your own proxy:
https://github.com/zhu327/gemini-openai-proxy
